onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -604,6 +604,13 @@ def data_type(self):
         '''
         return self._ortvalue.data_type()
 
+    def proto_type(self):
```

Comment:
Can we change the name to element_type or data_type? I think the proto_type is confusion. Same for other places.